core/src/test/java/org/apache/iceberg/TestDeleteFiles.java

```diff
@@ -68,38 +68,46 @@ public class TestDeleteFiles extends TableTestBase {
                   ))
           .build();
 
+  private final String branch;
+
   @Parameterized.Parameters(name = "formatVersion = {0}")
   public static Object[] parameters() {
-    return new Object[] {1, 2};
+    return new Object[][] {
+      new Object[] {1, "main"},
```

Comment:
I am little confused why formatVersion parameters were removed.

Reply:
That's correct, we test for every combination of format version + branch. I'll need update the @Parameters so that we can get nicer test names. The second parameter gets injected as an argument in the test file constructor.